src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/utils/JwtAuthHelper.kt

```diff
@@ -15,7 +14,7 @@ import java.util.UUID
 
 @Component
 class JwtAuthHelper {
-  private val keyPair: KeyPair
+  private final var keyPair: KeyPair
```

Comment:
we've also improved the code in https://github.com/ministryofjustice/hmpps-prisoner-location-api/blob/main/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerlocationapi/config/JwtAuthHelper.kt#L17 to get rid of the `init` and avoid the intellij warnings

Reply:
I used your example.